ci-operator/config/opendatahub-io/odh-dashboard/opendatahub-io-odh-dashboard-main.yaml

```diff
@@ -0,0 +1,46 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.17-openshift-4.10
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: odh-dashboard-image
+promotion:
+  namespace: opendatahub-io
+  tag: latest
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 400Mi
+tests:
+- as: odh-dashboard-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: odh-dashboard
```

Comment:
Tests are failing because cannot find this image, it should be `odh-dashboard-image` (line 9).

Reply:
Updated